packages/ckeditor5-package-generator/lib/utils/set-global-name.js

```diff
@@ -0,0 +1,40 @@
+/**
+ * @license Copyright (c) 2020-2024, CKSource Holding sp. z o.o. All rights reserved.
+ * For licensing, see LICENSE.md.
+ */
+
+'use strict';
+
+const { prompt } = require( 'inquirer' );
+const validateGlobalName = require( './validate-global-name' );
+
+/**
+ * Sets global name for generated package. It's used in UMD builds.
+ * If `--global-name` option us used, and it has valid value, that is returned.
+ * Otherwise, ask user to input the name.
+ *
+ * @param {Logger} logger
+ * @param {String} globalName
+ * @returns {Promise<String>}
```

Comment:
This is still JSDoc...

```suggestion
 * @returns {Promise.<String>}
```